### PR TITLE
Feat remove env vars

### LINKED ISSUE #179: remove dependency on environment variables
we should add default cli flags that can accommodate this so we dont need to explicitly set envs up front
AWS_PROFILE
AWS_REGION

### PR DESCRIPTION
**Pull request by converge**
fix #179

### CODE REVIEW

File: cmd/init.go [hunk 1]
```diff
@@ -97,6 +99,15 @@ to quickly create a Cobra application.`,
 		}
 		log.Println("region:", region)
 
+		profile, _ := cmd.Flags().GetString("profile")
+		viper.Set("aws.profile", profile)
```
Comment (6za) [inline cmd/init.go:103]: Should we be checking for error here? as the other ones..

Comment (6za) [reply cmd/init.go:103]: I saw that is later..

File: cmd/init.go [hunk 2]
```diff
@@ -97,6 +99,15 @@ to quickly create a Cobra application.`,
...
+		// propagate it to local environment
```
Comment (6za) [inline cmd/init.go:104]: Should we check for error here?
profile, _ := cmd.Flags().GetString("profile")

```golang
profile, err := cmd.Flags().GetString("profile")
		if err != nil {
			log.Println(err)
		}
```

Comment (converge) [reply cmd/init.go:104]: its always nice to be ready the day it will go wrong xD

**Review by pagottoo — APPROVED** [review]
/lgtm

**Pull request merged**